site-content/source/modules/ROOT/pages/development/documentation.adoc

```diff
@@ -8,18 +8,17 @@
 The official Cassandra documentation lives in the project's git
 repository.
 We use a static site generator, http://www.antora.org/[Antora], to create pages hosted at
-link:/doc/latest/[cassandra.apache.org].
-
+https://cassandra.apache.org/docs/latest/[cassandra.apache.org].
 <!-- You'll also find developer-centric content about Cassandra internals in our
 retired https://wiki.apache.org/cassandra[wiki] (not covered by this
 guide). -->
 
 Using a static site generator often requires the use of a markup language
 instead of visual editors (which some people would call good news).
-Antora processes http://www.asciidoc.org[Ascidoc], the markup language used to generate our documentation.
+Antora processes http://www.asciidoc.org[AsciiDoc], the markup language used to generate our documentation.
 Markup languages allow you to format text using certain syntax elements.
 Your document structure will also have to follow specific conventions.
-Feel free to take a look at http://cassandra.apache.org/docs[existing documents] to get a better idea how we structure our documents.
+Feel free to take a look at https://cassandra.apache.org/doc/latest/[existing documents] to get a better idea how we structure our documents.
```

Comment:
Please remove the `latest/` suffix, as agreed [here](https://github.com/apache/cassandra-website/pull/143#discussion_r904164956).
```suggestion
Feel free to take a look at https://cassandra.apache.org/doc/[existing documents] to get a better idea how we structure our documents.
```